blog/Getting Started with ANP.md

```diff
@@ -0,0 +1,270 @@
+  policies created by a single user (the one deploying the application).
+
+---
+
+Enter AdminNetworkPolicy API to the rescue. Designed specifically for cluster admins,
+AdminNetworkPolicy API provides two cluster-level resources that can not only enforce
+inviolable policy rules (for traffic that should always be allowed or denied), but also
+provide cluster default guardrails that can be overridden by namespace users if needed.
+Follow us along with an example below, as we showcase how the AdminNetworkPolicy API
+achieves this, and how it works along with the current NetworkPolicy objects to provide
+a complete, priority-based policy enforcing mechanism inside a K8s cluster.
+
+![Diagram1](Diagram1.png)
+
+Imagine we have a bare-bones cluster set up by a company's IT team, which only has a
+handful of namespaces in it. The sales team owns two namespaces `s1` and `s2` (both
+labeled tenant=sales), which runs web and db applications respectively. On the other
+hand, the accounting team owns a single namespace `a1`, which, for demonstration purposes,
+only has a single pod deployed. In addition, there is a `monitoring-ns` running an
+application that needs to probe into namespaces required by compliance team. Of course
+there is also `kube-system` who provides DNS services for the cluster via its `kube-dns`
+pods.
+
+With the help of the new API, admins can easily fulfill the networking requirements
+mentioned above, by applying the following `AdminNetworkPolicy` and
+`BaselineAdminNetworkPolicy` yaml specs at the cluster level:
+
+```yaml
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: cluster-basic-allow
+spec:
+  priority: 5
+  subject:
+    namespaces: {}
+  ingress:
+  - action: Allow   # Allows connection #4 labeled in the diagram
+    from:
+    - namespaces:
+        namespaceSelector:
+          matchLabels:
+            kubernetes.io/metadata.name: monitoring-ns
+  egress:
+  - action: Allow   # Allows connection #3 labeled in the diagram
+    to:
+    - namespaces:
+        namespaceSelector:
+          matchlabels:
+            kubernetes.io/metadata.name: kube-system
+      pods:
+        podSelector:
+          matchlabels:
+            app: kube-dns
+```
+
+```yaml
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: BaselineAdminNetworkPolicy
+metadata:
+  name: default
+spec:
+  subject:
+    namespaces:
+      namespaceSelector:  # Selects all non-kube-system Namespaces in the cluster
+        matchExpressions:
+          - {key:  kubernetes.io/metadata.name, operator: NotIn, values: [kube-system]}
+  egress:
+  - action: Allow         # Allows connection #1 labeled in the diagram
+    to:
+    - namespaces:
+        sameLabels: tenant
+  - action: Deny          # Drops connection #2 and #5 labeled in the diagram
+    to:
+    - namespaces:
+        namespaceSelector: {}
+```
+
+The syntax for the AdminNetworkPolicy API, as you can see, is very similar to that of the
```

Comment:
I think I would put this ANP overview section right at the top of the ANP bit before you introduce any of the diagrams or yamls.  It seems like a good introduction.